In [9]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt


service = pd.read_csv("service request.tsv", sep="\t")
service.head()

SR_NUM  INSPECT_ID                STREET          ADDRESS_2  \
0  HCE-20121221-7879      114434  442  WILLIAM STREET   AKA 441 HAYS AVE.   
1  HCE-20130206-2739      114441  918  GERRITT STREET                 NaN   
2  HCE-20130226-2887      114442      809  8TH STREET                 NaN   
3  HCE-20130226-2881      114443      801  8TH STREET                 NaN   
4  HCE-20130329-3145      114445  145  VICTORIA DRIVE                 NaN   

  APT_FLOOR            CITY STATE      ZIP  CENSUS_TRACT  \
0        #1    MOUNT OLIVER    PA  15210.0       54810.0   
1       NaN      PITTSBURGH    PA  15208.0       31207.0   
2       NaN  STOWE TOWNSHIP    PA  15136.0       54626.0   
3       NaN  STOWE TOWNSHIP    PA  15136.0       54626.0   
4       NaN       BRENTWOOD    PA  15227.0       54781.0   

            REQUEST_TYPE  PROPERTY_TYPE  NUM_UNITS EMERGENCY_TYPE  \
0      Complaint Housing   MULTI FAMILY        2.0  NON EMERGENCY   
1  Community Environment  SINGLE FAMILY        1.0  NON EMERGENCY   
2  Community Environment   MULTI FAMILY        3.0  NON EMERGENCY   
3  Community Environment   MULTI FAMILY        4.0  NON EMERGENCY   
4      Complaint Housing   MULTI FAMILY        4.0  NON EMERGENCY   

   INSPECTOR_ID  
0            18  
1            88  
2            88  
3            88  
4            46

In [10]:
service['REQUEST_TYPE']

0            Complaint Housing
1        Community Environment
2        Community Environment
3        Community Environment
4            Complaint Housing
                 ...          
24504    Community Environment
24505    Community Environment
24506        Complaint Housing
24507        Complaint Housing
24508        Complaint Housing
Name: REQUEST_TYPE, Length: 24509, dtype: object

The Data above describes the types of Service Request for the 24508 people who inputted their info.

In [7]:
service = service['REQUEST_TYPE'].value_counts()
service.head(15)

Complaint Housing            12891
Community Environment        10184
West Nile                     1384
Rodent                          44
Community Environment (8)        5
Other Vector                     1
Name: REQUEST_TYPE, dtype: int64

The Data above shows the service request that were called for the most from largest to smallest

In [16]:
service = service['REQUEST_TYPE'].value_counts().iloc[::-1]
service.head(25)

Other Vector                     1
Community Environment (8)        5
Rodent                          44
West Nile                     1384
Community Environment        10184
Complaint Housing            12891
Name: REQUEST_TYPE, dtype: int64

The data above starts from smallest to biggest with the amount of service requests that were called.

FINAL THOUGHTS
In summary the data summarizes the issue and request in certain neighborhoods. This data also shows the most common problemn with housing complaints and the environment and how the neighboorhood needs to be improved.